nautobot/extras/views.py

```diff
@@ -666,30 +667,56 @@ class CustomFieldBulkDeleteView(generic.BulkDeleteView):
 #
 
 
-class CustomLinkListView(generic.ObjectListView):
-    queryset = CustomLink.objects.all()
-    table = tables.CustomLinkTable
-    filterset = filters.CustomLinkFilterSet
-    filterset_form = forms.CustomLinkFilterForm
-    action_buttons = ("add",)
+class CustomLinkFieldsPanel(ObjectFieldsPanel):
+    def render_value(self, key, value, context):
+        if key == "target_url":
+            return pre_tag(value)
 
+        if key == "button_class" and value:
+            return format_html('<button class="btn btn-sm btn-{}">{}</button>', value.lower(), value.lower())
```

Comment:
Comparing to the existing template, it wasn't using `btn-sm` before. What's the reason for adding that now?